uber/models/attendee.py

```diff
@@ -749,6 +749,12 @@ def is_not_ready_to_checkin(self):
     def can_abandon_badge(self):
         return not self.amount_paid and not self.paid == c.NEED_NOT_PAY and not self.is_group_leader
 
+    @property
+    def can_refund_badge(self):
```

Comment:
Should this name be more specific? Like `can_self_service_refund_badge`? I assume that admins will still be able to refund the badge even if `can_refund_badge` is false.

Reply:
hmm... yea, that would probably be better, since 'refund' is a more ambiguous action word than 'abandon.'